src/app/shared/components/custom-table/custom-table.component.ts

```diff
@@ -9,6 +9,8 @@ import {
 } from '@angular/core';
```

Comment:
check all methods of the file. If any method is just for the file make it private and put it at the end of the file